charts/home-assistant/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 0.118.3
 description: Home Assistant
 name: home-assistant
-version: 3.0.0
+version: 4.0.0
```

Comment:
Why did you make this a major version bump? It's a bugfix on 3.0.0 if you ask me... 

The values should have already been in the values.yaml in 3.0.0, since the postgresql chart requires them there. If we would have put the values in the correct spot in our own values files it would have worked, we are only providing the correct defaults now.